components/merkledb/src/lib.rs

```diff
@@ -159,28 +159,23 @@
 pub use self::{
     backends::{rocksdb::RocksDB, temporarydb::TemporaryDB},
     db::{Database, DatabaseExt, Fork, Iter, Iterator, Patch, ReadonlyFork, Snapshot},
-    entry::Entry,
     error::Error,
-    group::Group,
     hash::{root_hash, HashTag, ObjectHash, ValidationError},
-    key_set_index::KeySetIndex,
+    indexes::{
+        Entry, Group, KeySetIndex, ListIndex, MapIndex, ProofEntry, SparseListIndex, ValueSetIndex,
+    },
     keys::BinaryKey,
     lazy::Lazy,
-    list_index::ListIndex,
-    map_index::MapIndex,
     options::DbOptions,
-    proof_entry::ProofEntry,
-    sparse_list_index::SparseListIndex,
-    value_set_index::ValueSetIndex,
     values::BinaryValue,
     views::{AsReadonly, IndexAddress, IndexType, ResolvedAddress, SystemSchema},
 };
 // Workaround for 'Linked file at path {exonum_merkledb_path}/struct.ProofMapIndex.html
 // does not exist!'
 #[doc(no_inline)]
 pub use self::{
-    proof_list_index::{ListProof, ProofListIndex},
-    proof_map_index::{MapProof, ProofMapIndex, RawProofMapIndex},
+    indexes::proof_list_index::{ListProof, ProofListIndex},
+    indexes::proof_map_index::{self, MapProof, ProofMapIndex, RawProofMapIndex},
```

Comment:
Why is `self` re-exported for `proof_map_index`, but it not for `proof_list_index`?

Reply:
For clarity and backward compatibility. From `proof_list_index` module we need to export only `ProofListIndex` and `ListProof`.  From `proof_map_index` we need to export `Raw`, `Hashed`, `ProofPath` and so on.  And I think it's more clear to use `proof_map_index::Raw` than `merkledb::Raw`.

Reply:
Hm... I'm a bit concerned about asymmetry here, but the explanation seems to be reasonable. OK then.